tests/test_smoke.py

```diff
@@ -2227,7 +2227,7 @@ def test_aws_zero_quota_failover():
 
     test = Test('aws-zero-quota-failover', [
         f'sky launch -y -c {name} --cloud aws --region {region} --gpus V100:8 --use-spot | grep "Found no quota"',
-    ])
+    ], f'sky down -y {name}')
```

Comment:
nit: add a `,` at the end to make the format align with the other tests.
```suggestion
    ], f'sky down -y {name}',)
```